pennylane_lightning_kokkos/src/simulator/StateVectorKokkos.hpp

```diff
@@ -516,8 +521,10 @@ template <class Precision> class StateVectorKokkos {
      *
      * @param num_qubits Number of qubits
      */
-    StateVectorKokkos(Kokkos::complex<Precision> *hostdata_, size_t length)
-        : StateVectorKokkos(Util::log2(length)) {
+    StateVectorKokkos(
+        Kokkos::complex<Precision> *hostdata_, size_t length,
+        Kokkos::InitArguments kokkos_args = {})
```

Comment:
Is there any benefit to doing
```suggestion
        const Kokkos::InitArguments& kokkos_args = {})
```
instead of a copy construction?

Reply:
Yes, as mentioned above, save a bit of memory and time (for each instance created, which can pile up).